src/ontology/mondo-edit.obo

```diff
@@ -159781,6 +159790,7 @@ xref: OMIM:147050 {source="MONDO:equivalentTo"}
 xref: UMLS:C1840253 {source="MONDO:equivalentTo", source="MONDO:MEDGEN", source="MEDGEN:327063"}
 is_a: MONDO:0003847 {source="https://orcid.org/0000-0001-5208-3432"} ! hereditary disease
 is_a: MONDO:0004980 ! atopic eczema
+relationship: excluded_from_qc_check http://purl.obolibrary.org/obo/mondo/sparql/qc/mondo/qc-omim-subsumption.sparql
```

Comment:
I am surprised this is not failing as a single child?

I also wonder if 'IgE responsiveness, atopic' should be a child of 'atopic eczema'? There is no source for that. Should it be a child of 'hypersensitivity reaction disease' (based on the definition?)

Reply:
I am surprised the single child qc did not fail as well. Let me review other possible parent terms....

Reply:
Moving to a new parent of 'hypersensitivity reaction disease' seems fine for now. I would prefer a new term 'Type  I hypersensitivity disease', but see that MONDO:0025512 'type II hypersensitivity reaction disease' is marked for obsoletion so I am hesitant to create this new term if all 4 hypersensitivity types will eventually be removed.

Reply:
we have type 1 hypersenstivity: MONDO:0007817 'IgE responsiveness, atopic', it is an exact synonym

Reply:
Gotcha, I missed that when looking at the hierarchy since MONDO:0007817 'IgE responsiveness, atopic' is not a child of MONDO:0000605 'hypersensitivity reaction disease' so it looks like type I hypersensitivity reaction is missing. MONDO:0007817 'IgE responsiveness, atopic' would be a better parent.